### ISSUE TITLE
Change command to check kernel version in documentation

### ISSUE THREAD

**Issue by thomasdanan** [scality/metalk8s#2288]
**Component**:

'documentation', 'deployment'

**Why this is needed**:

Because of the cgroups memory leak bug in Linux kernel, it is requested to check the right kernel version is deployed. In the documentation, we suggest using the command `uname -r` in order to make sure the version is higher than `3.10.0-1062.4.1.el7.x86_64`. This command may be not enough precise depending on the OS.

**What should be done**:

Documentation should suggest using the command `rpm -q kernel` instead (in https://documentation.scality.com/metalk8s/2.4.2/installation/setup.html)

**Implementation proposal** (strongly recommended):

**Test plan**:

**Comment by NicolasT** [comment]
> Documentation should suggest using the command rpm -q kernel instead.

No, that's not sufficient. Having the package installed doesn't imply it's the version that's running.

**Comment by slaperche-scality** [comment]
What's the problem with `uname`?

IIRC, it uses the `uname` syscall and we can assume that the operating system presumably knows its name, release and version. (`rpm` doesn't look very portable too me, less than `uname` at least, but I guess it's fine…).

**Comment by NicolasT** [comment]
We got a report of a RHEL 7.7 system where `uname -r` said `3.10.0-1062.el7.x86_64`, which seems to provide less details than the `uname -r` on my current CentOS 7.7 system which returns `3.10.0-1062.12.1.el7.x86_64`.

**Comment by thomasdanan** [comment]
@NicolasT, then what is the right set of commands or the procedure to check we have the right kernel installed?

**Comment by NicolasT** [comment]
Well, it *should* be `uname -r`. Why the output we received has little details, I don't know... Maybe they *are* running a `3.10.0-1062.el7.x86_64` package on that system, and `3.10.0-1062.12.1.el7.x86_64` would be the one installed after updating said system? :man_shrugging:

**Comment by alexandre-allard** [comment]
We can still get the information from `/proc/sys/kernel/osrelease`, but I'm not sure there will be more information. Maybe uname also is getting the information from there.
Edit: Yes, look like the output is the very same (at least on Ubuntu Bionic & CentOS 7).
Edit2: Looks like there really is a Kernel 3.10.0-1062 (first Kernel version that came with RedHat 7.7, released on August 6, 2019).

**Comment by NicolasT** [comment]
The main question is: if on a RHEL system `uname -r` shows `3.10.0-1062.el7.x86_64`, is this 'lower' than `3.10.0-1062.4.1.el7.x86_64`, or would, say, a `3.10.0-1062.12.1` kernel still show up as `3.10.0-1062.el7.x86_64`?

**Comment by alexandre-allard** [comment]
It's lower, it corresponds to a version `3.10.0-1062.0.0`.
Give me a minute, I try on a RedHat.

**Comment by alexandre-allard** [comment]
It was a very long minute :)
So, I've tested, it is as I said in my last comment.
Kernel `3.10.0-1062` is the first version of RedHat Kernel on 7.7, so it corresponds to a `3.10.0-1062.0.0`.
If you upgrade to a newer version, like `3.10.0-1062.4.1.el7.x86_64`, return from `uname -r` will be what we expect.
So, no need to change the command, maybe we could just add few words to explain that `3.10.0-1062` is older than any ``3.10.0-1062.X.X``

**Comment by wabernat** [comment]
This change is entered in https://github.com/scality/metalk8s/pull/2304
